sql/moz-fx-data-shared-prod/telemetry_derived/clients_histogram_bucket_counts_v1/query.sql

```diff
@@ -44,13 +44,44 @@ static_combos AS (
     '*' AS os,
     '*' AS app_build_id
 ),
+build_ids AS (
```

Comment:
Was there a test for this one? Just wondering if there was a test that should have failed but didn't

Reply:
There is test for `histogram_probe_counts` and not for bucket_counts